php/add-subscriber-to-list.php

```diff
@@ -0,0 +1,80 @@
+<?php
+  /*
+   * Add a 'member' to a 'list' via mailchimp API v3.x
+   * @ http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#create-post_lists_list_id_members
+   *
+   * ================
+   * BACKGROUND
+   * Typical use case is that this code would get run by an .ajax() jQuery call or possibly a form action
+   * The live data you need will get transferred via the global $_POST variable
+   * That data must be put into an array with keys that match the mailchimp endpoints, check the above link for those
+   * You also need to include your API key and list ID for this to work.
+   * You'll just have to go get those and type them in here, see README.md
+   * ================
+   */
+
+  // Set API Key and list ID to add a subscriber
+  $api_key = 'your-api-key-here';
```

Comment:
Consider using constants as they should not change for the lifetime of the script. Example:
```
define('API_KEY', 'your-api-key-here');
```
and access with:
```
curl_setopt($ch, CURLOPT_USERPWD, 'user:' . API_KEY);
```